drush/Commands/DeployCommands.php

```diff
@@ -95,6 +95,95 @@ public function backstop(string $target, string $reference, array $options = ['c
     $this->logger()->success($this->getSuccessMessage($body));
   }
 
+  /**
+   * Run Backstop Snapshot at CircleCI and save the result for usage with ma:backstop-reference.
+   *
+   * @command ma:ci:backstop-snapshot
+   *
+   * @option target Target environment. Recognized values: prod, test, local, tugboat, feature[N].
+   * @option list The list you want to run. Recognized values: page, all, post-release. See backstop/backstop.js
+   * @option viewport The viewport you want to run.  Recognized values: desktop, tablet, phone. See backstop/backstop.js.
+   * @option ci-branch The branch that CircleCI should check out at start, default value is "develop"
+   * @usage drush ma:ci:backstop-snapshot --target=prod
+   *   Create a backstop snapshot from production.
+   * @aliases ma-ci-backstop-snapshot
+   * @validate-circleci-token
+   *
+   * @throws \Exception
+   * @throws \GuzzleHttp\Exception\GuzzleException
+   */
+  public function ci_backstop_snapshot(array $options = ['target' => 'prod', 'ci-branch' => 'develop', 'list' => 'all', 'viewport' => 'all']): void {
```

Comment:
@weitzman I don't know if you'd prefer this in a separate file or not, but I stuck it in here as it's where the existing backstop command is